### ISSUE TITLE
Light/Dark theme selector not present

### ISSUE THREAD

**Issue by MikhaD** [photodiode/panorama-view#124]
## Version: 0.9.1

After the latest update panorama view defaulted to light mode, and the option for dark mode no longer seems to be present in the options.

![image](https://user-images.githubusercontent.com/31388146/138136102-8b1ea9a9-4cf9-4867-b1ec-21b7f0d49329.png)

**Comment by photodiode** [comment]
What theme are you using?

I had a feeling the auto-picker would fail in some cases..

Right now it gets the current themes main background color and checks it's luminosity, but that threshold is a bit arbitrary.

**Comment by MikhaD** [comment]
I'm always in favor of giving the user more control, not taking it away. People who don't need it won't notice whether its there or not, the only people who will care when a feature is removed are the people who used it, so why not leave it in for the people who do like to have more control.
I don't think having the option to manually toggle takes away from your ideal of having it just slide into the current theme unnoticed.

I am new to the browser add on space, so please forgive me if I am misunderstanding, but would it not be possible to determine the users theme using the css media query @media (prefers-color-scheme: dark)?

**Comment by MikhaD** [comment]
Yes, it is the panorama view that isn't dark. Your solution sounds great.
When I have more time I'd like to try writing an add on. I started one a few years ago, and this has piqued my interest in it again.
